### PR TITLE
Fixes #2 Created XML layout for Login Activity 

### LINKED ISSUE #2: XML layout for Login Activity
## Description
Create a XML layout for login activity based on the design specifications given in [Readme](https://github.com/bsoc-bitbyte/DigiGate/blob/main/README.md#design) file.

### Points to be consider:
1. Use Constraint Layout.
2. The layout should follow the general design specifications with reasonable padding and margin values chosen by the contributor.

### Describe the solution you'd like

I would like to have an XML layout file (`res/layout/activity_login.xml`) for the Login Activity that includes all the necessary UI elements as described above. The layout should:

 The XML layout should be implemented in `res/layout/activity_login.xml`.
- The layout should match the general design specifications provided in the README file.
- The layout should use appropriate padding and margin values chosen by the contributor.
- The layout should be visually consistent with the design mockup provided in the README file.

### Describe alternatives you've considered

_No response_

### Developer Help

_No response_

### PR DESCRIPTION
**Pull request by madhavs004**
## Overview

1. This PR fixes or fixes part of #2 
2. This PR does the following: Created the XML layout for Login Activity and added the necessary fonts, icons and resources for custom buttons and edit texts into drawable.

## Essential Checklist

- [ x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [ x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [ x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot (480)

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

**Comment by aarav0012** [comment]
add  all the colors in colors.xml and strings in strings.xml then use it

**Review by aarav0012 — COMMENTED** [review]
add all the colors and strings in their respective file and use it from there

**Comment by madhavs004** [comment]
> add all the colors and strings in their respective file and use it from there

I have made the necessary changes and pushed the new commit into the PR.

File: app/src/main/res/values/colors.xml
Comment (aarav0012) [inline app/src/main/res/values/colors.xml:1]: every color should have one name, so modify it accordingly

File: app/src/main/res/layout/activity_login.xml
Comment (aarav0012) [inline app/src/main/res/layout/activity_login.xml:1]: dimensions should be in multiple of 4 (padding, margin, textsize)

**Comment by aarav0012** [comment]
after changes mention all dimension used on discord

**Comment by madhavs004** [comment]
> after changes mention all dimension used on discord
I have mentioned the changes on the Digigate discussion channel. Please review them from there .

File: app/src/main/res/values/colors.xml
Comment (madhavs004) [reply app/src/main/res/values/colors.xml:1]: I have pushed the commit in the PR after making the required changes.

**Comment by aarav0012** [comment]
LGTM

**Pull request merged**